source/elements/oneDAL/source/glossary.rst

```diff
@@ -130,9 +155,10 @@ Machine learning terms
         becomes available in parts over time.
 
     Reference-counted object
-        A |dal_short_name| object which stores the number of references to the unique implementation.
-        Such object has a lightweight copy constructor and assignment operator, which simply increases the number of references.
-        Implementation is automatically freed when the number of references becomes equal to zero.
+        A copy-constructible and copy-assignable |dal_short_name| object
```

Comment:
reference to the source that contains def of copy-constructible?

Reply:
Hmm... this is defined by the C++ standard, which is not freely available. I can add a reference to cppreference.com, which is a (non-official) reference for all terms and entities defined by C++ standard.
But do we really need that? This is a spec for C++/DPC++ library, so we can assume that the reader is familiar with C++ terms.

Reply:
okay, if the others think in the same direction